pcieutil/main.py

```diff
@@ -105,6 +108,150 @@ def pcie_show():
         Id = item["id"]
         click.echo("bus:dev.fn %s:%s.%s - dev_id=0x%s, %s" % (Bus, Dev, Fn, Id, Name))
 
+# Show PCIe AER status
+
+
+@cli.group(cls=clicommon.AliasedGroup)
+def pcie_aer():
+    '''Display PCIe AER status'''
+    pass
+
+
+@pcie_aer.command()
+def correctable():
+    '''Show PCIe AER correctable attributes'''
+
+    statedb = SonicV2Connector()
+    statedb.connect(statedb.STATE_DB)
+    header = ['AER - CORRECTABLE']
+
+    # AER - Correctable fields (9)
+    fields = ['RxErr', 'BadTLP', 'BadDLLP', 'Rollover', 'Timeout', 'NonFatalErr', 'CorrIntErr', 'HeaderOF', 'TOTAL_ERR_COR']
+    table = OrderedDict()
+    for field in fields:
+        table[field] = [field]
+
+    resultInfo = platform_pcieutil.get_pcie_check()
+    for item in resultInfo:
+        if item["result"] == "Failed":
```

Comment:
Shouldn't this be "Success"? Display the AER status for the Failed case not success case?

Reply:
AER stats will be displayed only for PCIe devices that pass 'get_pcie_check'